app/controllers/item.js

```diff
@@ -38,9 +37,9 @@ export default Ember.Controller.extend({
   }),
 
   hasDraftOrder: Ember.computed.alias("session.draftOrder"),
-  isOrderFulfilmentUser: Ember.computed(function() {
+  isOrderFulfilmentUser: Ember.computed('session.authToken', function() {
```

Comment:
This comouted property should only get executed if user is available. As I've said before it should only check whether User has order fulfillment role or not. It shouldn't check if the user is logged in or not

Reply:
If it is not executed after logged out, then 'Inventory number' remains visible on the screen.